src/Surfnet/StepupMiddleware/CommandHandlingBundle/DependencyInjection/CompilerPass/AddPipelineStagesCompilerPass.php

```diff
@@ -18,19 +18,45 @@
 
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\DependencyInjection\CompilerPass;
 
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\RuntimeException;
 use Symfony\Component\DependencyInjection\Reference;
 
 class AddPipelineStagesCompilerPass implements CompilerPassInterface
 {
+    /**
+     * {@inheritdoc} Since the priorities cannot be changed runtime but only through configuration, we're doing the
+     * sorting based on priority here. A lower priority means the stage is added earlier.
```

Comment:
> A lower priority means the stage is added earlier.

Isn't this counterintuitive? Eg. Symfony event listeners are executed from high to low priority. If ascending numbers are preferred, perhaps rename `priority` attribute to something else?